intake_esm/core.py

```diff
@@ -855,6 +855,14 @@ def to_dataset_dict(
             lon_bnds   (lon, bnds) float64 dask.array<chunksize=(320, 2), meta=np.ndarray>
             time_bnds  (time, bnds) object dask.array<chunksize=(1980, 2), meta=np.ndarray>
             pr         (member_id, time, lat, lon) float32 dask.array<chunksize=(1, 600, 160, 320), meta=np.ndarray>
+
+        Usage Notes
+        -----------
```

Comment:
```suggestion
        Notes
        -----
```

Reply:
Instead of adding a `Notes` section, we want to update the existing description of `cdf_kwargs`
https://github.com/intake/intake-esm/blob/00fe97676b165bf41fb985a5e03b07a4a9772b71/intake_esm/core.py#L816-L817

to something along these lines:

```python
Keyword arguments to pass to :py:func:`~xarray.open_dataset` function. The specified chunking dictionary should refer to dimensions that are present in a single netcdf file. Attempting to chunking along dimensions that are created during concatenation of multiple datasets will result into a failure.
```

Reply:
OK, thanks for clarifying.   Was your comment suggesting that "chunks" is an expected parameter for cdf_kwargs?   Because I may choose to replace "When" with "If" otherwise.